src/connector/type_identifier.rs

```diff
@@ -0,0 +1,13 @@
+pub(crate) trait TypeIdentifier {
+    fn is_real(&self) -> bool;
+    fn is_integer(&self) -> bool;
+    fn is_datetime(&self) -> bool;
+    fn is_time(&self) -> bool;
+    fn is_date(&self) -> bool;
+    fn is_text(&self) -> bool;
+    fn is_bytes(&self) -> bool;
+    fn is_bool(&self) -> bool;
+    fn is_json(&self) -> bool;
+    fn is_enum(&self) -> bool;
+    fn is_null(&self) -> bool;
+}
```

Comment:
:100:

Reply:
I tried to use these with pg, but it doesn't really fit there that well...